query-service-impl/src/main/java/org/hypertrace/core/query/service/multivalue/FunctionTransformation.java

```diff
@@ -41,6 +46,9 @@ public Single<Function.Builder> transformFunction(Function.Builder builder) {
   private Single<Function.Builder> transformDistinctCount(Function.Builder functionBuilder) {
     return this.getFirstAttributeId(functionBuilder)
         .flatMap(this.attributeClient::get)
+        .doOnError(
+            error -> log.error("Unable to get resolve attribute using attribute service", error))
+        .onErrorReturnItem(DEFAULT_ATTRIBUTE_METADATA)
```

Comment:
Why are we defaulting?

Reply:
mostly for integration tests to run

Reply:
Then shouldn't we fix the integration test? will check with you, but I can review other parts.

BTW, the pinot integration test might be failing due to - https://github.com/hypertrace/view-generator-framework/pull/67 (There was client version change, and I am not fully sure, but it seems it has some issue)

Reply:
This change is reverted